test/integration/rules/aria-toggle-field-label/aria-toggle-field-label.html

```diff
@@ -0,0 +1,87 @@
+<!-- PASS -->
+<!-- checkbox -->
+<div id="pass1" aria-label="newspaper" role="checkbox">Newspaper</div>
+<!-- menuitemcheckbox -->
+<ul role="menu">
+	<li
+		id="pass2"
+		role="menuitemcheckbox"
+		aria-label="Word wrap"
+		aria-checked="true"
+	>
+		Word wrap
+	</li>
+</ul>
+<!-- menuitemradio -->
+<ul role="menu">
+	<li
+		id="pass3"
+		role="menuitemradio"
+		aria-label="Sans-serif"
+		aria-checked="true"
+	>
+		Sans-serif
+	</li>
+</ul>
+<!-- radio -->
+<div role="radiogroup">
+	<div
+		id="pass4"
+		role="radio"
+		aria-checked="false"
+		tabindex="0"
+		aria-label="Regular crust"
+	>
+		Regular crust
+	</div>
+</div>
+<!-- switch -->
+<div
+	id="pass5"
+	role="switch"
+	aria-checked="true"
+	aria-label="Display blue light:"
+>
+	<span>off</span>
+	<span>on</span>
+</div>
+
+<!-- FAIL -->
```

Comment:
All these test cases are wrong. These elements should all pass because they have an accessible name from content.